gcp/appengine/frontend3/src/styles.scss

```diff
@@ -598,6 +598,7 @@ dl.vulnerability-details,
   grid-template-columns: max-content minmax(0, auto);
   grid-row-gap: 5px;
   gap: 8px;
+  margin-bottom: 15px;
```

Comment:
Very nice! 

Super nit: any chance we can use https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child to not add the extra whitespace in the last entry?

Reply:
Done, added a not(last-child) selector for .vulnerability-package-subsection dl.